technische-universitat-dresden-medienwissenschaft-und-neuere-deutsche-literatur-note.csl

```diff
@@ -180,14 +185,9 @@
       </choose>
       <choose>
         <if type="bill book graphic legal_case motion_picture report song" match="any">
-          <choose>
-            <if variable="volume">
-              <group delimiter=" ">
-                <text term="volume" form="short"/>
-                <number variable="volume"/>
-              </group>
-            </if>
-          </choose>
+          <group delimiter=" " suffix=".">
+            <text macro="volume"/>
+          </group>
```

Comment:
@jmiba, groups with just a single child element can always be replaced by just the child element, e.g.

```
          <text macro="volume" suffix="."/>
```

instead of

```xml
          <group delimiter=" " suffix=".">
            <text macro="volume"/>
          </group>
```